tensorflow_serving/apis/load_model_prediction_service.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package tensorflow.serving;
+option cc_enable_arenas = true;
+
+import "tensorflow_serving/apis/get_model_metadata.proto";
+import "tensorflow_serving/apis/predict.proto";
+
+// open source marker; do not remove
+// PredictionService provides access to machine-learned models loaded by
+// model_servers.
+service LoadModelPredictionService {
+  // Predict -- provides access to TensorFlow model and loads if required.
+  rpc LoadPredict(PredictRequest) returns (PredictResponse);
```

Comment:
Would you consider a different protobuf for `LoadPredict` request please? i.e.

```
message LoadPredictRequest {
  PredictRequest predict_request = 1;
  string model_base_path = 2;
}
```

This will allow the caller to specify the `model_base_path`, i.e. somewhere in S3 or local filesystem mounted with EFS.

Reply:
If you do decide to give this a go, then we can put the `LoadPredictRequest` in `apis/load_model_prediction.proto` file to be consistent with other `.proto` files.